deny-not-main-or-master-revision/allow.rego

```diff
@@ -0,0 +1,8 @@
+package env0
+
+# METADATA
+# title: allow by default
```

Comment:
`title` is not needed or used here.

Reply:
[acea3ec](https://github.com/env0/approval-policies/pull/5/commits/acea3ec9f19baac981f37c75a5190d80c09581df)